server/src/routes/business/send-mail.js

```diff
@@ -46,12 +53,24 @@ export const sendMailToAccount = async (candidat, flag) => {
   return sendMail(candidat.email, message)
 }
 
-export const sendMagicLink = async (candidat, token) => {
-  const flag = 'CHECK_OK'
-  const authUrl = `${config.PUBLIC_URL}${config.CANDIDAT_ROUTE}`
+export const sendMagicLink = async candidat => {
+  const flag = AURIGE_OK
 
-  const url = `${authUrl}?token=${encodeURIComponent(token)}`
+  const url = getUrl(candidat)
 
   const message = await getMailData(candidat, flag, url)
   return sendMail(candidat.email, message)
 }
+
+export const getUrl = candidat => {
+  const candidatAccessDate = getFrenchLuxonFromJSDate(candidat.canAccessAt)
+  const dateNow = getFrenchLuxon()
+
+  if (!candidatAccessDate || dateNow > candidatAccessDate) {
+    const token = createToken(candidat.id, config.userStatuses.CANDIDAT)
+    const authUrl = `${config.PUBLIC_URL}${config.CANDIDAT_ROUTE}`
+    return `${authUrl}?token=${encodeURIComponent(token)}`
+  }
+
+  return null
```

Comment:
Est-ce vraiment une bonne idée ?
Cypress ressort une erreur: `You're attempting to visit this new domain: 'null'`, je pense que c'est lié...